recipes/kalpy/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "kalpy" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: kalpy
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/k/kalpy-kaldi/kalpy-kaldi-{{ version }}.tar.gz
+  sha256: 43cf28d35a97751d4f1b40f133897c67f66dfd81bb0d8d010c826998cca2b49e
+
+build:
+  script:
+    - CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY" {{ PYTHON }} -m pip install . --no-deps -vv  # [not win]
+  number: 0
+  skip: True  # [py<38]
+  script_env:
+   - KALDI_ROOT={{ environ.get('LIBRARY_PREFIX') }}  # [win]
+   - KALDI_ROOT={{ environ.get('PREFIX') }}  # [not win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - python                                  # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}      # [build_platform != target_platform]
+    - cmake
+    - ninja
+    - m2-patch  # [win]
+    - libtool   # [not win]
+    - automake  # [not win]
+    - make  # [not win]
+    - pybind11
+    - setuptools
+    - setuptools_scm
+  host:
+    - python
+    - pip
+    - kaldi * cpu*
```

Why does this depend on the CPU builds? What would go wrong if someone tried to use the CUDA builds? FTR, I don't think we should constrain this.
```suggestion
    - kaldi
```

---

Yeah that makes sense, I was getting some errors when I didn't constrain it, but I can try another pass with them now that I've fixed the other blocking errors.

---

It's still there in the host requirements...